scripts/run_several_instances.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
a script to stop all these binaries would be nice too

Reply:
Modified the script so ctrl-c will kill all the services. So now we can run the script, and if we just want to kill everything we just do ctrl-c and everything will be instantly killed.